pkg/alert/config.go

```diff
@@ -96,6 +96,10 @@ func BuildAlertmanagerConfig(address string, timeout time.Duration) (Alertmanage
 
 	scheme := parsed.Scheme
 	host := parsed.Host
+	if scheme == "" || host == "" {
+		return AlertmanagerConfig{}, errors.New("alertmanagers.url contains empty scheme or host")
+	}
+
```

Comment:
Optional: We could separate this into separate conditions so that we can tell user exactly which part is missing.